src/style.scss

```diff
@@ -24,9 +24,9 @@
 //   src: url('./fonts/Roboto-Regular.ttf') format('truetype');
 // }
 
-::-webkit-scrollbar {
-  display: none;
-}
+// ::-webkit-scrollbar {
```

Comment:
Why is this commented out?

Reply:
for displaying the scrollbar 
ABX-10931

Reply:
Let's delete it then